index.js

```diff
@@ -57,10 +59,32 @@ var PersonaClient = function (config) {
         }
     }
 
-    // connect to redis and switch to the configured db
-    var redis = require('redis');
-    this.redisClient = redis.createClient(this.config.redis_port, this.config.redis_host);
-    this.redisClient.select(this.config.redis_db);
+    var CacheService = require("cache-service");
```

Comment:
Just a personal style preference but I'm a fan of having all the non-dynamic requires at the top of the file.